cms/static/js/views/instructor_info.js

```diff
@@ -0,0 +1,100 @@
+// Backbone Application View: Instructor Information
+
+define([  // jshint ignore:line
+        'jquery',
+        'underscore',
+        'backbone',
+        'gettext',
+        'js/utils/templates',
+        "js/models/uploads",
+        "js/views/uploads"
+    ],
+    function ($, _, Backbone, gettext, TemplateUtils, FileUploadModel, FileUploadDialog) {
+        'use strict';
+        var InstructorInfoView = Backbone.View.extend({
+
+             events : {
+                 'click .remove-instructor-data': 'removeInstructor',
+                 'click .action-upload-instructor-image': "uploadInstructorImage"
+             },
+
+            initialize: function(options) {
+                // Set up the initial state of the attributes set for this model instance
+                 _.bindAll(this, 'render');
+                this.template = this.loadTemplate('course-instructor-details');
+                this.model = options.model;
```

Comment:
Nit: the model variable should get set up for you automatically (I've never had to explicitly set it).